### ISSUE TITLE
"func.running_mean" doesn't equal to the corrsponding "base.bn1.running_mean" in resnet?

### ISSUE THREAD

**Issue by zouliangyu** [longcw/pytorch2caffe#10]
hi, I have tranformed a caffe model from a pytorch model, after transformation, I compared all the correspoding parameters between caffe and pytorch. Conv layer's(weights, bias) and BN layer's scale parameters are all right, but the running_mean, running_var in pytorch are not corresponding to base.bn1.running_mean, base.bn1.running_var, may I ask do you know why, thank you @longcw !!!

**Comment by longcw** [comment]
Did you add `model.eval()` like this https://github.com/longcw/pytorch2caffe/blob/master/pytorch2caffe.py#L394.
The running_mean will be updated in the training mode.

**Comment by zouliangyu** [comment]
@longcw Oh yes, THANKS, I forgot to add it to fix the mean and var ...
